Vwitter/UserCell.m

```diff
@@ -0,0 +1,102 @@
+//
+//  UserCell.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import "UserCell.h"
+#import "Follow.h"
+
+@implementation UserCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
+    [super setSelected:selected animated:animated];
+
+    // Configure the view for the selected state
+}
```

Comment:
nit: These methods can actually be removed. All they do is call the same method on `super` (the super class implementation of the same method). The only time you would override/implement these methods is when you have your own custom logic that needs to take place in addition to the "default" logic for what happens when, for example, the cell is selected (`setSelected:animated:`)